analytics/src/analytics/integrations/db.py

```diff
@@ -25,11 +25,15 @@ def get_db() -> Engine:
     db = get_db_settings()
     # inspired by simpler-grants-gov/blob/main/api/src/adapters/db/clients/postgres_client.py
     token = db.password if db.local_env is True else generate_iam_auth_token(db)
+    url = f"postgresql+psycopg://{db.user}:{token}@{db.db_host}:{db.port}?sslmode={db.ssl_mode}"
+    print(f"TEMP DEBUG: db settings = {db}")
+    print(f"TEMP DEBUG: token = {token}")
```

Comment:
```suggestion
```

We can't have this in the logs, even if it's temporary

Reply:
@coilysiren ok, I removed it